org-code-javabuilder/lib/src/main/java/org/code/javabuilder/JavaRunner.java

```diff
@@ -43,22 +43,28 @@ public JavaRunner(
    *     finished executing.
    */
   public void runMain() throws InternalFacingException, JavabuilderException {
-    this.run(this.mainRunner);
+    this.run(this.mainRunner, false);
   }
 
   public void runTests() throws JavabuilderException, InternalFacingException {
-    this.run(this.testRunner);
+    // Tests have more permissions than a regular run--as of now, all
```

Comment:
I'm not sure if this was a typo or intentional - you have two dashes (--)

Reply:
it was intentional, meant to be an [em dash](https://www.grammarly.com/blog/dash/), probably should be a colon instead for clarity

Reply:
👍